main/webapp/modules/core/scripts/index/parser-interfaces/xml-parser-ui.js

```diff
@@ -257,6 +260,11 @@ Refine.XmlParserUI.prototype._setRecordPath = function(path) {
 
 Refine.XmlParserUI.prototype._updatePreview = function() {
   var self = this;
+  
+  if(!self._recordElementSelected) {
+    alert("No record element selected! Please select one first.");
```

Comment:
Oh, and that's clearly something we want to localize using i18n. And perhaps avoid the exclamation mark in the message.

Reply:
Hang on, we already have this message: `core-index-import/warning-record-path`.

Reply:
This seems to duplicate the error message here: https://github.com/OpenRefine/OpenRefine/pull/2407/files#diff-92cdef36b4cf54a220fa4a4a7acc5107R65